include/libstatistics_collector/moving_average_statistics/moving_average.hpp

```diff
@@ -52,6 +53,9 @@ class MovingAverageStatistics
   LIBSTATISTICS_COLLECTOR_PUBLIC
   MovingAverageStatistics() = default;
 
+  LIBSTATISTICS_COLLECTOR_PUBLIC
+  explicit MovingAverageStatistics(const std::size_t & window_size);
```

Comment:
```suggestion
  explicit MovingAverageStatistics(std::size_t  window_size);
```

Reply:
Watch out for the extra space:

```suggestion
  explicit MovingAverageStatistics(std::size_t window_size);
```

And we could keep the `const`.

Reply:
For primitives types, we should always pass by value, not reference.